apps/entry/serializers.py

```diff
@@ -418,6 +418,21 @@ class Meta:
         exclude = ('comment',)
 
 
+class AssignmentEntryCommentSerializer(serializers.ModelSerializer):
+    content_id = serializers.IntegerField(source='id', read_only=True)
+    content_title = serializers.CharField(source='text', read_only=True)
+    project = serializers.IntegerField(source='entry.project_id', read_only=True)
+    project_title = serializers.CharField(source='entry.project.title', read_only=True)
+    created_by_name = serializers.CharField(source='created_by.username', read_only=True)
+    entry_excerpt = serializers.CharField(source='entry.excerpt', read_only=True)
+
+    class Meta:
+        model = EntryComment
+        fields = ('content_id', 'content_title', 'entry',
+                  'project', 'project_title', 'created_by',
+                  'created_by_name', 'entry_excerpt')
```

Comment:
Let's define this in notification/serializer.py.

Also, we don't need to define project fields here, let's use the assignment project column directly.